src/main/java/com/dev/cinema/model/dto/CinemaHallRequestDto.java

```diff
@@ -1,7 +1,12 @@
 package com.dev.cinema.model.dto;
 
+import com.sun.istack.NotNull;
```

Comment:
```suggestion
import javax.validation.constraints.NotNull;
```
same for all requestDtos